pkg/utils/k8sutil.go

```diff
@@ -181,19 +181,20 @@ func GetFunctionData(runtime, ftype, modName string) (imageName, depName, fileNa
 	fileName = ""
 
 	type runtimeVersion struct {
+		runtimeID   string
 		version     string
 		httpImage   string
 		pubsubImage string
 	}
 
-	python27 := runtimeVersion{version: "2.7", httpImage: python27Http, pubsubImage: python27Pubsub}
+	python27 := runtimeVersion{runtimeID: "python", version: "2.7", httpImage: python27Http, pubsubImage: python27Pubsub}
 	python := []runtimeVersion{python27}
 
-	node6 := runtimeVersion{version: "6", httpImage: node6Http, pubsubImage: node6Pubsub}
-	node8 := runtimeVersion{version: "8", httpImage: node8Http, pubsubImage: node8Pubsub}
+	node6 := runtimeVersion{runtimeID: "node", version: "6", httpImage: node6Http, pubsubImage: node6Pubsub}
+	node8 := runtimeVersion{runtimeID: "node", version: "8", httpImage: node8Http, pubsubImage: node8Pubsub}
 	node := []runtimeVersion{node6, node8}
 
-	ruby24 := runtimeVersion{version: "2.4", httpImage: ruby24Http, pubsubImage: ""}
+	ruby24 := runtimeVersion{runtimeID: "ruby", version: "2.4", httpImage: ruby24Http, pubsubImage: ""}
 	ruby := []runtimeVersion{ruby24}
 
```

Comment:
I think these above variables (python, node, ruby) are unchanged (the images ref might be changed) and can be initialized at the beginning so they should be global variables and put in the init function